### PR TITLE
Fix version component margin and darktheme background for status bar

### PR DESCRIPTION
**Pull request by Asizon**
image

image

### CODE REVIEW

**Comment by Asizon** [comment]
@chmelevskij do you think we should go with this `!important` fix?Seems that now components go before dark theme

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by chmelevskij — DISMISSED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by Asizon** [comment]
I will update this pr to just darktheme fix.

**Review by haslinghuis — APPROVED** [review]

**Review by McGiverGim — APPROVED** [review]

**Review by limonspb — APPROVED** [review]

**Review by chmelevskij — APPROVED** [review]

**Pull request merged**
